airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcSqlGenerator.kt

```diff
@@ -237,54 +251,40 @@ constructor(
             condition =
                 condition.and(
                     DSL.field(DSL.name(JavaBaseConstants.COLUMN_NAME_AB_EXTRACTED_AT))
-                        .gt(minRawTimestamp.get().toString())
+                        .gt(formatTimestampLiteral(minRawTimestamp.get())),
                 )
         }
         return condition
     }
 
-    override fun createSchema(schema: String?): Sql {
+    override fun createSchema(schema: String): Sql {
         return of(createSchemaSql(schema))
     }
 
     override fun createTable(stream: StreamConfig, suffix: String, force: Boolean): Sql {
         // TODO: Use Naming transformer to sanitize these strings with redshift restrictions.
         val finalTableIdentifier = stream.id.finalName + suffix.lowercase(Locale.getDefault())
         if (!force) {
-            return transactionally(
-                Stream.concat(
-                        Stream.of(
-                            createTableSql(
-                                stream.id.finalNamespace,
-                                finalTableIdentifier,
-                                stream.columns!!
-                            )
-                        ),
-                        createIndexSql(stream, suffix).stream()
-                    )
-                    .toList()
+            return of(
+                createTableSql(stream.id.finalNamespace, finalTableIdentifier, stream.columns)
```

Comment:
do we really want ot drop the index? Seems a bit dangerous. I'm probably missing something